docs/DeveloperGuide.md

```diff
@@ -305,6 +305,57 @@ the filtered transaction with a success message.
   - 2a1. UniCa$h displays an error message with the correct command format.
   - Use case resumes at step 1.
 
+**Use Case: UC11 - Set Budget**
+
+**MSS:**
+1. User enters the command to set the budget with the correct format (i.e. specifying the amount and interval).
+2. User submits the request.
+3. UniCa$h sets the global budget.
+4. UniCa$h displays the new budget.
+
+   Use Case ends.
+
+**Extensions:**
+
+- 1a. User enters the incorrect format (missing fields)
+  - 1a1. UniCa$h displays an error message with the correct command format
+  - Use case resumes at step 1.
```

Comment:
Should the entering incorrect format fall under 2a. (Referring to how the other Use cases are done above)

Reply:
I think it doesn't make sense for it to start with 2a if step 1 was already that the user entered the command in the right format

Reply:
That makes sense, we should probably update the rest too.

Reply:
Yup, I can make a PR o pick this up